FordLabs/retroquest
# Re-implement drag and drop

## Overview
This PR re-implements drag and drop for retro thoughts into the react version of retroquest.

Connects N/A

### What was done
- [x] Added drag and drop to retro thoughts using [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd)
- [x] Wrote cypress tests for drag and drop
- [x] Removed current animation implementation for adding and removing retro items. We'll readdress at a later date.  

## Testing Instructions
1) Try dragging and dropping retro items from column to column with your mouse. Ensure they move and the border color changes to the new column color.
2) Try dragging and dropping retro items from column to column with your keyboard. You can do this by tabbing to a retro item, pressing the spacebar, then pressing the right or left arrow keys, and pressing the spacebar key again to "drop" the item.
